Automattic/jetpack
# Legacy Widgets: Return Stats Widgets for Simple Sites

Follow-up to #38564

## Proposed changes:

The API for the Blog Stats and Top Posts and Pages block is currently broken on Simple sites (see #36141). As such, the block has been hidden for them in #38564. As the block is unavailable, however, it probably makes sense to ensure that they can still use the Legacy Widget given that the block was intended to replace this. 

## Jetpack product discussion


See #38564.

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Activate the Extra Sidebar Widgets module in Jetpack.
* On a classic theme, navigate to Widgets and confirm that "Blog Stats" and "Top Posts and Pages" still appear in the Legacy Widget on a Simple site.
* These two widgets should not appear on other sites - those sites can use the new and enhanced blocks, for which the endpoint still works.
